src/Umbraco.Web/PropertyEditors/MultiUrlPickerConfiguration.cs

```diff
@@ -11,13 +10,12 @@ public class MultiUrlPickerConfiguration : IIgnoreUserStartNodesConfig
         [ConfigurationField("maxNumber", "Maximum number of items", "number")]
         public int MaxNumber { get; set; }
 
-        [ConfigurationField(Core.Constants.DataTypes.ReservedPreValueKeys.IgnoreUserStartNodes,
-            "Ignore User Start Nodes", "boolean",
+        [ConfigurationField(Core.Constants.DataTypes.ReservedPreValueKeys.IgnoreUserStartNodes, "Ignore user start nodes", "boolean",
             Description = "Selecting this option allows a user to choose nodes that they normally don't have access to.")]
         public bool IgnoreUserStartNodes { get; set; }
 
-        [ConfigurationField("hideAnchor",
-            "Hide anchor/query string input", "boolean",
+        // TODO Change this to hideQueryString, as that's the name used in the picker
+        [ConfigurationField("hideAnchor", "Hide anchor/query string input", "boolean",
```

Comment:
This configuration key doesn't correspond to the Multi URL Picker name, both `LinkDisplay` and `LinkDto` saves it as `queryString`... Although the linkpicker overlay calls it `anchor` again 🤔

Reply:
Let's keep this the way it is, as changing the name would cause a breaking change.